pages.pt_BR/common/2to3.md

```diff
@@ -0,0 +1,32 @@
+# 2to3
+
+> Conversão automática de código Python 2 para Python 3.
+> Mais informação: <https://docs.python.org/3/library/2to3.html>.
+
+- Mostrar as alterações que seriam feitas sem faze-las de fato (simulação):
+
+`2to3 {{diretorio/do/arquivo.py}}`
+
+- Converter um arquivo feito em Python 2 para Python 3:
+
+`2to3 --write {{diretorio/do/arquivo.py}}`
+
+- Converter recurso específico de Python 2 para Python 3:
+
+`2to3 --write {{diretorio/do/arquivo.py}} --fix={{raw_input}} --fix={{print}}`
+
+- Converter todos os recursos de Python 2 para Python 3, exceto as que específicadas:
+
+`2to3 --write {{diretorio/do/arquivo.py}} --nofix={{has_key}} --nofix={{isinstance}}`
+
+- Mostrar a lista de todas os recursos disponíveis que podem ser convertidas de Python 2 para Python 3:
+
+`2to3 --list-fixes`
+
+- Converter todos os arquivos feitos em Python 2 em um diretório para Python 3:
+
+`2to3 --output-dir={{diretorio/que_contem/arquivos_python3}} --write-unchanged-files --nobackups {{diretorio/que_contem/arquivos_python2}}`
```

Comment:
I don't like the main page for this example:
```
--output-dir=path/to/python3_directory
```
sound like the installation directory for python3. :thinking:

Reply:
Hey! I also thought a bit like how you thought, that's why I changed it in the translation.

`diretorio/que_contem/arquivos_python3` means something like `path/that_contains/python3_files`

Reply:
@navarroaxel that's a good point.  @sbrl do you think `path/that_contains/python3_files` is fine?

Reply:
I checked this in Ubuntu focal (Docker) and `more v2.34` needs the `+10` before the file.

Reply:
What about `python3_output`?

Reply:
Ping @pedrokpp / @bl-ue